inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1107,6 +1113,17 @@ bool layout_optimizer::are_data_types_suitable_for_onednn(program_node& node) {
     return false;
 }
 
+bool layout_optimizer::are_layouts_suitable_for_onednn(program_node& node) {
+    auto in_layout = node.get_dependencies().front()->get_output_layout();
+    auto out_layout = node.get_output_layout();
+    if (node.get_preferred_impl_type() == impl_types::onednn && node.is_type<convolution>() &&
```

Comment:
It is not only for convolution, but the entire onednn primitives.

Reply:
Is it valid for reorder as well?